src/gui/guiFormSpecMenu.cpp

```diff
@@ -738,10 +832,12 @@ void GUIFormSpecMenu::parseImage(parserData* data, const std::string &element)
 			1
 		);
 		core::rect<s32> rect(pos, pos + geom);
-		gui::IGUIImage *e = Environment->addImage(rect, this, spec.fid, 0, true);
+		gui::IGUIImage *e = Environment->addImage(rect, data->current_parent,
+				spec.fid, 0, true);
 		e->setImage(texture);
 		e->setScaleImage(true);
-		e->setNotClipped(true);
+		if (data->current_parent == this)
+			e->setNotClipped(true);
```

Comment:
How about styles? Probably needs rebase?

Reply:
I've rebased, there were no conflicts (and I couldn't find any new place where `this` used as parent).
#9262 will probably cause a conflict here.
:thinking: This `if` here was probably a bad idea. When #9262 is merged, the not clipping will only be the default for old formspec versions (which don't support `scroll_container`s anyway). This will make the change here unneeded, so I'll remove it.